src/game/boe.newgraph.cpp

```diff
@@ -399,6 +399,8 @@ void do_missile_anim(short num_steps,location missile_origin,short sound_num) {
 	// Now, at last, launch missile
 	for(short t = 0; t < num_steps; t++) {
 		draw_terrain();
+		// prevent drawing to happen outside the terrain windows
+		clip_rect(mainPtr, terrain_screen_rect);
```

Comment:
Unfortunately this doesn't compile after cherry-picking it… I assume it depends on some other commit that I didn't take…

Reply:
This is the commit: 8c5938d12081f57f0d20f11ead9a5bcee2a40c38